scipy/optimize/tests/test__sda.py

```diff
@@ -0,0 +1,120 @@
+# Simulated dual annealing unit tests implementation.
+# Copyright (c) 2017 Sylvain Gubian <sylvain.gubian@pmi.com>,
+# Yang Xiang <yang.xiang@pmi.com>
+# Author: Sylvain Gubian, PMP S.A.
+##############################################################################
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+##############################################################################
+"""
+Unit tests for the simulted dual annealing global optimizer
+"""
+from .._sda import sda
```

Comment:
These imports should be `from scipy.optimize import sda`. We want to test the exact interface that a user will see.

Reply:
Fixed.

Reply:
Good point! Thanks!